xml/repeated-content-decl.ent

```diff
@@ -0,0 +1,118 @@
+<!--...........................................................................
+Repeated content entities, for re-printing content in multiple locations
+rather than using xrefs, so that users do not have to skip around so much
+This works just like copy-paste, so be mindful of sections, titles, and
+paragraphs
+............................................................................-->
+
+<!-- Use xmlns="http://docbook.org/ns/docbook" on the first occurrence of every
+formatting tag in each entity, to avoid "Namespace default prefix was not found" 
+errors -->
+
+<!ENTITY namespaces 
+'<formalpara xmlns="http://docbook.org/ns/docbook">
+<title>Length of release names</title> 
+<para xmlns="http://docbook.org/ns/docbook">
+Release names (e.g. when you run <command>helm install --name</command>) 
+have a maximum length of 36 characters.
+</para>
+</formalpara>
+<formalpara xmlns="http://docbook.org/ns/docbook">
+<title>Always install to a fresh namespace</title>
+<para xmlns="http://docbook.org/ns/docbook">
+    If you are not creating a fresh &productname; installation, but have deleted 
+    a previous deployment and are starting over, you must create new namespaces
+    with the 
+    <command>kubectl create namespace <replaceable>name</replaceable></command>
+        command. 
+    Do not re-use your old namespaces. The <command>helm delete</command> command 
+    does not remove generated secrets from the SCF and UAA namespaces as it is 
+    not aware of them. These leftover secrets will cause problems. See 
+    <xref linkend="sec.cap.rebuild.depl"/> for more information.
+    </para>
+    </formalpara>'>
+    
+<!ENTITY readmefirst
+  '<important xmlns="http://docbook.org/ns/docbook">
+      <title xmlns="http://docbook.org/ns/docbook">README first!</title>
+      <para xmlns="http://docbook.org/ns/docbook">
+       Before you start deploying &productname;, review the following documents:
+      </para>
+      <itemizedlist xmlns="http://docbook.org/ns/docbook">
```

Comment:
Same here: no need for the DocBook namespace on `title`, `para` and `itemizedlist`.